### DISCUSSION TITLE
Flexible sidebar info drawer

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#4351] [Feature Requests]
It would be nice if you could drag the left-side of the drawer to resize it manually (doesn't need to save to preferences). It should also have a min/max to keep things working smoothly.

**Comment by philleepflorence** [comment]
Wonderful idea!
It seems there is already a maximum according to the [Reference: Directus Vue Components - Drawer.](https://docs.directus.io/reference/app/components/v-drawer/)

**PS: How do we know when a discussion is being developed?**

**Reply by benhaynes** [reply]
When we start to work on something convert it from a Discussion to an Issue. If an external contributor starts working on something they can let us know here, and once there's a PR to review, we can move it to an Issue too.

**Reply by benhaynes** [reply]
Also, remember to upvote Discussions that you want to see implemented... that's how we track the most "requested" features.

**Reply by philleepflorence** [reply]
Thanks for the confirmation!

So an **external contributor** can leave a **comment** on a **discussion** should they wish to **implement**?

**Reply by benhaynes** [reply]
Yup!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
